src/app/pages/private/LifeListPage/displayLife.tsx

```diff
@@ -0,0 +1,28 @@
+import { useListLivesQuery } from '../../../api';
+import NoItemResult from '../../../components/results/NoItemResult';
+
+const DisplayLife = () => {
+    const { data, loading } = useListLivesQuery();
+
+    if (loading) {
+        // not yet loaded
+        return null;
+    }
+
+    const badam = data?.lives || [];
+    console.log(badam)
+    if (badam.length === 0) {
+        return <NoItemResult />
+    }
+
+    const DisplayData = () => {
+        return (
+            <>
+            {badam.map((life) => <p>{life.id} {life.title} {life.fullName}</p>)}
```

Comment:
If your `ReactNode` is an `array` then very top-level element should have a unique `key` property for React Virtual DOM to properly pick on updates. Usually you will have the `key` value to be an UID specific to your item, if none is available then the index as fallback solution.